api/swagger/swagger.yaml

```diff
@@ -0,0 +1,429 @@
+          $ref: '#/responses/badRequest'
+        403:
+          $ref: '#/responses/forbidden'
+        404:
+          $ref: '#/responses/notFound'
+    put:
+      tags:
+        - user
+      summary: Update user
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+      - name: username
+        in: path
+        description: The name that needs to be fetched.
+        required: true
+        type: string
+      responses:
+        200:
+          $ref: "#/responses/ok"  
+        400:
+          $ref: '#/responses/badRequest'
+        403:
+          $ref: '#/responses/forbidden'
+    delete:
+      tags:
+        - user
+      summary: Delete user
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+      - name: username
+        in: path
+        description: The name that needs to be fetched.
+        required: true
+        type: string
+      responses:
+        200:
+          $ref: "#/responses/ok"  
+        400:
+          $ref: '#/responses/badRequest'
+        403:
+          $ref: '#/responses/forbidden'
+  /login:
+    get:
+      tags:
+        - auth
+      summary: Logs user into the system
+      produces:
+        - application/json
+      parameters:
+        - name: username
+          in: query
+          description: The user name for login
+          required: true
+          type: string
+        - name: password
+          in: query
+          description: The password for login in clear text
+          required: true
+          type: string
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/User'
+          description: Successful operation
+        400:
+          $ref: '#/responses/badRequest'
+  /logout:
+    get:
+      tags:
+        - auth
+      summary: Logs out current logged in user session
+      produces:
+        - application/json
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/User'
+          description: Successful operation
+        400:
+          $ref: '#/responses/badRequest'
+  /stats:
+    get:
+      tags:
+        - statistic
+      summary: Get posts and comments statistics
+      produces:
+        - application/json
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/Statistics'
+          description: Posts and comments statistics
+        400:
+          $ref: '#/responses/badRequest'
+definitions:
+  User:
+    type: object
+    properties:
+      id:
+        type: integer
+        format: int64
+      username:
+        type: string
+      firstName:
+        type: string
+      lastName:
+        type: string
+      email:
+        type: string
+      password:
+        type: string
+      phone:
+        type: string
+  Post:
+    type: object
+    required:
+      - id
+      - createdAt
+      - postBody
+    properties:
+      id:
+        type: integer
```

Comment:
so it's not uuid? https://github.com/makacies/nodejs-practice/pull/1/files#diff-a9245a482d8363d9f4d3a821f1beb9d4R201

Reply:
yes, it should be uuid, I forgot to update it, I'll fix it